modules/hostap/src/supp_main.c

```diff
@@ -696,6 +696,14 @@ static int hostapd_disable_iface_cb(struct hostapd_iface *hapd_iface)
 	supplicant_send_wifi_mgmt_ap_status(hapd_iface,
```

Comment:
Please fix multiple typos in commit message:
`s/pharse/phrase/`

Also no need to have `zephyr:` prefix in commit subject, it is redundant information.

Reply:
done

Reply:
@jukkar , please help review the PR, thanks